Graeae.Models.Tests/ApiTests.cs

```diff
@@ -0,0 +1,16 @@
+using Graeae.Models.SchemaDraft4;
+
+namespace Graeae.Models.Tests;
+
+public class ApiTests
+{
+    [Test]
+    public void MultipleEnableDoesNotThrow()
+    {
+        Assert.DoesNotThrow(() =>
+        {
+            Draft4Support.Enable();
+            Draft4Support.Enable();
+        });
+    }
```

Comment:
We don't need to call this twice within the method because the `TestEnvironment` class already call it before any test methods are run.

```suggestion
    public void InitializingAgainDoesNotThrow()
    {
        Draft4Support.Enable();
    }
```

Also, I never understood the explicit `DoesNotThrow()` method.  I think it's unnecessary.  If it doesn't throw, the test passes; if it throws, the test fails.